<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SyncJson</class>
 <widget class="QMainWindow" name="SyncJson">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_files">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>9</y>
      <width>621</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>File Info.</string>
    </property>
    <widget class="QPushButton" name="pushButton_tar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Open Target Json</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_tar">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>50</y>
       <width>471</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>수정할 대상 Json 파일을 Open합니다.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_src">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>471</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>값을 가져올 Json 파일을 Open합니다.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_src">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Open Source Json</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_cmp">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Compare</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_clear">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>80</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_save">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>80</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Save as</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_content">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>621</width>
      <height>301</height>
     </rect>
    </property>
    <property name="title">
     <string>Contents</string>
    </property>
    <widget class="QTableWidget" name="tableWidget_cmp">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>601</width>
       <height>271</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>파일(F)</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menu_F"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit(Q)</string>
   </property>
   <property name="shortcut">
    <string>Q</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
